src/components/views/dialogs/AddExistingToSpaceDialog.tsx

```diff
@@ -65,6 +66,7 @@ export const Entry: React.FC<{
                 onChange={onChange ? (e) => onChange(e.currentTarget.checked) : undefined}
                 checked={checked}
                 disabled={!onChange}
+                aria-labelledby={id}
```

Comment:
This shouldn't be needed for an `input` inside a `label`

Reply:
Ah right, I remember why I did this now (should really leave a comment)

![image](https://github.com/user-attachments/assets/2e80264d-74d0-44b9-ae72-8bd0ff134814)

Currently the avatar's are being included in the label which gives a confusing name. At first I thought I could just mark the avatar as role=presentation to avoid it being included, but that didn't work. I've gone for reformatting the Entry component to instead be a list element, and explicitly marked the `room.name` as a label.

There is also now a test.